library/src/containers/Schemas/NewSchema.tsx

```diff
@@ -0,0 +1,345 @@
+import React, { useState } from 'react';
+import { Schema } from '@asyncapi/parser';
+
+import { Chevron, Markdown } from '../../components';
+import { SchemaHelpers } from '../../helpers';
+
+interface Props {
+  schemaName?: string;
+  schema?: Schema;
+  required?: boolean;
+  isCircular?: boolean;
+  odd?: boolean;
```

Comment:
`odd` looks weird to me. What does it mean that a prop is odd? (I know what it means, just want you to think about it from a different perspective).

Should this be called something like `darkBackground` or `lightBackground` or something like this? Something that refers more to what it actually does.

Reply:
Maybe this light/dark background thingy can be handled outside this component? For instance, the NewSchema component is transparent and the color of the background is handled in the parent component.

Reply:
The `odd` I "copied" from HTML template and you probably understood what it is - it "checks" that another schema renders in another nested schema like `oneOf` or `object` as prop in another `object`. Of course we can call this `nested` or something like that, but HTML template do it in proper way - odd indicated also how nested schema is. `darkBackground` or `lightBackground` is good, but we should choose only one. 

> Maybe this light/dark background thingy can be handled outside this component? For instance, the NewSchema component is transparent and the color of the background is handled in the parent component.

The main problem with it are very nested schemas and style "wrappers" for them. Transparent of schema should works only for 'root' schema, not for nested. I will try to check something like:

```css
.schema > .schema {
 // light styles
}

.schema > .schema > .styles {
 // dark styles
}
```

Reply:
Ok, I removed `odd` prop and we make "odd" styles by pure css combinators.